static/templates/recipient_row.handlebars

```diff
@@ -9,13 +9,13 @@
                 title="{{#tr this}}Narrow to stream &quot;__display_recipient__&quot;{{/tr}}">
                 {{! invite only lock }}
                 {{#if invite_only}}
-                <i class="fa fa-lock invite-stream-icon" aria-hidden="true" title="{{t 'This is a private stream' }}"></i>
+                <i class="fa fa-lock invite-stream-icon" title="{{t 'This is a private stream' }}" aria-label="{{t 'This is a private stream' }}"></i>
                 {{/if}}
                 {{display_recipient}}
             </a>
 
             {{! hidden narrow icon for copy-pasting }}
-            <span class="copy-paste-text">&gt;</span>
+            <span class="copy-paste-text" role="button" tabindex="0" aria-label="{{ t 'Copy paste' }}">&gt;</span>
```

Comment:
This should be hidden for ARIA; it's HTML content that's only visible to copy-paste tools, not a button.